pages/index.js

```diff
@@ -1,27 +1,14 @@
+import { Catalog } from '@/components/catalog'
 import { Layout } from '@/components/layout'
-import { Info } from '@carbonplan/icons'
-import { Box, Flex, Text } from 'theme-ui'
+import { Box, Container } from 'theme-ui'
 
 export default function Home() {
   return (
     <Layout>
-      <Box
-        sx={{
-          display: 'flex',
-          justifyContent: 'center',
-          alignItems: 'center',
-        }}
-      >
-        <Flex>
-          <Info />
-          <Text
-            sx={{
-              px: 2,
-            }}
-          >
-            Site under Construction
-          </Text>
-        </Flex>
+      <Box>
+        <Container>
+          <Catalog />
+        </Container>
```

Comment:
It's really helpful to add the `Guide` from `@carbonplan/components` to help debug elements that are not adhering to the grid structure (more info https://carbonplan.org/design/grid). It might be worth adding the `Guide` here and use it get all elements onto the grid.
```suggestion
        <Container>
          <Guide />
          <Catalog />
        </Container>
```

<img width="1204" alt="CleanShot 2023-04-04 at 09 25 44@2x" src="https://user-images.githubusercontent.com/12436887/229856423-8705c475-30b9-40ad-b5f0-b8d23a257b5c.png">

Reply:
To view the grid structure in the mocks in Figma, you can press `ctrl + g` to reveal the layout grid.

Reply:
![](https://media1.giphy.com/media/OK27wINdQS5YQ/giphy.gif?cid=ecf05e47yh4wzhptslunhnrncrhm8pfy0y0v0mpxn43q54cc&rid=giphy.gif&ct=g)

Woah!! i've always wondered how you revealed the layout grid during some of our pair programming sessions. 

thank you for this tip!